### DISCUSSION TITLE
Set ```MFXTreeItem``` Expand(Chevron) Icon to right from left.

### DISCUSSION THREAD

**Discussion by infinite-dev22** [palexdev/MaterialFX#356] [Q&A] [answered]
I want to check the location/position of the Arrow(Chevron) on the MFXTreeItem from left to right.
From this:
![Screenshot from 2023-11-28 22-11-53](https://github.com/palexdev/MaterialFX/assets/77667007/b2949990-d27c-4cde-973d-3663a073cd3b)
To this:
![Screenshot from 2023-11-28 22-10-12](https://github.com/palexdev/MaterialFX/assets/77667007/be4c26aa-650d-42db-97a3-b352a205b9b4)

Is there a way I can achieve this, Thanks in advance.

**Comment by palexdev** [comment, marked as answer]
Absolutely.
You need a custom cell, check the code of [MFXSimpleTreeCell](https://github.com/palexdev/MaterialFX/blob/main/materialfx/src/main/java/io/github/palexdev/materialfx/controls/cell/MFXSimpleTreeCell.java)
The "disclosure Node" is handled there, you just need to change the code so that it is always positioned at the end
Beware multiple methods need to be changed even `render` probably

**Reply by infinite-dev22** [reply]
Thank your for the timely response Sir, let try that out.

**Reply by infinite-dev22** [reply]
Hello, Good day Sir.
I was able to pool this off(Though not yet as perfect).
I have failed to style the ```MFXTreeView```, does it use a Virtual flow.
Also I tried styling the virtual flow on the tree view but failed. How can I achieve this, Thanks.
